_config.yml

```diff
@@ -9,6 +9,7 @@ url: "http://developer.fedoraproject.org" # the base hostname & protocol for you
 twitter_username: jekyllrb
 github_username:  jekyll
 encoding: utf-8
+fedora_release: 34
```

Comment:
We can access the latter variable through `page` like this: `{{ page.fedora_release }}`, so this is not needed
```suggestion
```